alexandria/core/serializers.py

```diff
@@ -273,7 +264,6 @@ class Meta:
         fields = BaseSerializer.Meta.fields + (
             "variant",
             "name",
-            "original",
```

Comment:
Revert. Otherwise it will not appear in the response as seen in the snapshots.

Read only fields must be declared in both `fields` **and** `read_only_fields`

Reply:
good catch, fixed in other PR.